src/spec-node/dockerCompose.ts

```diff
@@ -705,6 +705,7 @@ export function dockerComposeCLIConfig(params: Omit<PartialExecParameters, 'cmd'
 		return result || (result = (async () => {
 			let v2 = false;
 			let stdout: Buffer;
+			dockerCLICmd='docker-compose';
```

Comment:
`function dockerComposeCLIConfig` already accepts `dockerComposeCLICmd` as parameter Can you instead update all the places where this function is called? eg. https://github.com/devcontainers/cli/blob/58e0a470e19473514af63a0d9ea0ff4a488de3cb/src/spec-node/devContainers.ts#L165C36-L165C53

Reply:
`dockerComposeCLICmd` is passed in from the `--docker-compose-path` CLI option. I think we need to keep the CLI option specify the Docker Compose v1 path to not break existing setups. The existing `--docker-path` (passed in here as `dockerCLICmd`) implies the Docker Compose v2 path.

I think we need to:
- Swap the two calls to 'version'.
- Start with `v2 = true`.
- Remove the check for `ENOENT`? Since `docker` likely exists, but if v2 is missing then `docker compose` will return an error.